conf/vmware.yaml.template

```diff
@@ -1,33 +1,31 @@
 VMWARE:
-  # Vmware to be added as a compute resource
-  # vmware vcenter URL, e.g. example.com
+  # VMware to be added as a compute resource
+  # VCENTER: vmware vcenter URL
```

Comment:
Can you update all the examples which @lhellebr suggested?

Reply:
I've already updated all examples as Uppercase